paying_invoices.md

```diff
@@ -0,0 +1,27 @@
+[Back](https://mrets.github.io/Help/index)
+
+# Where do I submit payment?
+
+Invoice Payment Instructions:
+
+  1. Fees must be paid in US dollars\
+  2. Payments may be made via credit card, wire transfer, ACH transfer or check\
```

Comment:
Updated this to contain the change from https://github.com/mrets/Help/pull/136/files which was the source of the conflict.